pom.xml

```diff
@@ -251,6 +251,18 @@
       <version>9.1-901-1.jdbc4</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
+      <version>1.7.1</version> <!-- Stay on 1.7.1 to support Java 6 -->
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>eu.codearte.catch-exception</groupId>
+      <artifactId>catch-exception</artifactId>
```

Comment:
Some of the junit rules needed switched to something.  So this was required in a number of cases.  In fact, I ended up just clearing all junit rules entirely out.  One major benefit to using this method is that catch exception proxies the exception so code coverage on unit test looks better.  That library isn't necessary once on java 8 though.  For now just needed something to fill the gap.  Although direct usage of try/catch with reading the exceptions would have also worked.